util/util_test.go

```diff
@@ -255,6 +255,16 @@ func TestHasOwner(t *testing.T) {
 			},
 			expected: true,
 		},
+		{
```

Comment:
nit: Can we add a test where MachineSet is owned by v1alpha2 MachineDeployment alone?

Reply:
Done ptal

Reply:
Thanks!

Reply:
This test could be added to `TestMachineSetOwnerReference` too.